From cfebb0f03359dd12f771e5e61fb6672acb34a4e1 Mon Sep 17 00:00:00 2001
From: Yong Zhao <yong.zhao@amd.com>
Date: Tue, 2 Jan 2018 15:10:18 -0500
Subject: [PATCH 5729/5855] drm/amdkfd: Delete an useless call of
 kfd_flush_tlb()

During process creation, no vmid is associated with the process for both
HWS and non-HWS case, because no queues have been created. So a TLB
flush doesn't make sense.

Change-Id: I7e7a170cf4b095b74b24555f923a7d6b0d09b929
Signed-off-by: Yong Zhao <yong.zhao@amd.com>
---
 drivers/gpu/drm/amd/amdkfd/kfd_process.c | 2 --
 1 file changed, 2 deletions(-)

diff --git a/drivers/gpu/drm/amd/amdkfd/kfd_process.c b/drivers/gpu/drm/amd/amdkfd/kfd_process.c
index e8e4ddb..e178dda 100644
--- a/drivers/gpu/drm/amd/amdkfd/kfd_process.c
+++ b/drivers/gpu/drm/amd/amdkfd/kfd_process.c
@@ -136,8 +136,6 @@ static int kfd_process_alloc_gpuvm(struct kfd_process *p,
 		goto sync_memory_failed;
 	}
 
-	kfd_flush_tlb(kdev, p->pasid);
-
 	/* Create an obj handle so kfd_process_device_remove_obj_handle
 	 * will take care of the bo removal when the process finishes.
 	 * We do not need to take p->mutex, because the process is just
-- 
2.7.4

